pkg/pillar/types/global.go

```diff
@@ -19,6 +19,12 @@ const (
 	symlinkDir       = TmpDirname + "/GlobalConfig"
 )
 
+// GlobalStatus - Status of Global Conig Items.
+type GlobalStatus struct {
+	// Any Errors seen in Config Items
```

Comment:
Why not define the key/value part here as well?
We should fill those in so that the controller can see all of the keys and values a particular device has.
I don't know if it makes sense to defer those until the restructuring of the configItem code.

Reply:
With the restructure - the GlobalConfig structure will change into a Map. So what ever code we write now will be a waste. Once we have the new structure, it is just a map that we need to copy over. Hence waiting for that.
Archismaan has already started on that. So it is not far away.

I would recommend waiting for it. For this change, lets just get the errors in - that is the painpoint point being addressed in the review anyways..

Reply:
You can't copy a map since it ends up being a copy of a pointer. I think you need to create each status entry from the config entry, and then add that to the status map to make sure you
1. don't accidentally just point at the config
2. don't include elements with unknown keys

But this can be deferred to Archismaan changes as long as he is aware that he is the one to do it.